packages/wrangler/src/__tests__/cloudchamber/deploy.test.ts

```diff
@@ -346,6 +346,113 @@ describe("wrangler deploy with containers", () => {
 		expect(std.warn).toMatchInlineSnapshot(`""`);
 	});
 
+	it("resolve the dockerfile based on the non-redirected userConfigPath rather than the actual config path", async () => {
```

Comment:
```suggestion
	it("resolve the dockerfile based on the non-redirected userConfigPath rather than the redirected config path", async () => {
```
🤔 ?